test/mpi/coll/allred.c

```diff
@@ -379,107 +398,258 @@ struct double_test {
 #define test_types_set5(op, post) do { } while (0)
 #endif
 
-static int test_allred(int count, mtest_mem_type_e evenmem, mtest_mem_type_e oddmem)
+static void test_types_set2_sum_1()
```

Comment:
these all should have `void` argument, right? strict builds might complain.

Reply:
I could fix them. But if you don't mind, maybe it is simpler to just squash the commits so these temporary routines goes away?

Reply:
Squash is fine. We're looking at ~1000 lines of diff total, but its all to a single file which was basically impossible to read in the first place...